src/index.js

```diff
@@ -0,0 +1,71 @@
+import './style.css';
+import getAllSeaFoodMeals from './modules/meals.js';
+import { addLikeOnEvent, populateCurrentLikes } from './modules/cardlikes.js';
+import logo from '../assets/logo-kan.png';
+import like from '../assets/like2.png';
+import getItem from './modules/reservation.js';
+
+const logoImg = document.querySelector('.logo');
+logoImg.src = logo;
+
+let allSeafoods = [];
+
+(async () => {
+  try {
+    allSeafoods = await getAllSeaFoodMeals();
+  } catch (error) {
+    /* eslint-disable */
+    console.error(error.message);
+    /* eslint-enable */
+  }
```

Comment:
Hello, @Ghiftee thank you so much for the detailed review, however, could you please suggest what could be an alternative here to get the message in case an error occurs? [in this particular case inside a catch block]